dotcom-rendering/src/components/ShareButton.importable.tsx

```diff
@@ -189,6 +189,12 @@ export const ShareButton = ({
 	const isLiveBlogMeta =
 		format.design === ArticleDesign.LiveBlog && context === 'ArticleMeta';
 
+	const isLiveBlogBlockDesktop =
+		typeof window !== 'undefined' &&
+		format.design === ArticleDesign.LiveBlog &&
+		context === 'LiveBlock' &&
```

Comment:
I suspect if you want to target liveblog blocks then `context === 'LiveBlock'` is likely to be enough since they don't appear outside liveblogs?

```suggestion
		context === 'LiveBlock' &&
```